src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4393,6 +4393,16 @@ namespace ZoneEquipmentManager {
             ShowFatalError("DistributeSystemOutputRequired: Illegal load distribution scheme type.");
             break;
         }
+        // For every load distribution scheme except SequentialLoad
+        //  set the remaining loads to the first equipment type's load to support equipment types that don't use the sequenced loads
```

Comment:
Hmm, looking at this now, this may not be a general fix - what if the equipment that doesn't use sequenced loads is not first in the sequence of equipment?